ui/core-react/src/core-react/tree/Node.tsx

```diff
@@ -141,7 +141,9 @@ export class TreeNode extends React.Component<TreeNodeProps> {
     ) : undefined;
 
     const toggle =
-      this.props.isLoading || this.props.isLeaf ? undefined : (
+      this.props.isLoading || this.props.isLeaf ? (
+        <div />
```

Comment:
Looks like this empty `div`  causes some alignment issues when loading spinner is shown.
![image](https://github.com/iTwin/appui/assets/24278440/66ea4404-0350-43cd-acf6-30d698e0beab)

Reply:
Empty div should no longer appear when loading is occurring